From dd3282f27eadb2f3de1c6910895c3834163a958c Mon Sep 17 00:00:00 2001
From: Jonathan Nieder <jrnieder@gmail.com>
Date: Mon, 26 Sep 2011 16:30:15 -0500
Subject: [SHELL] Disable sh -c "command" -> sh -c "exec command" optimization

It is causing ocamlbuild and the darcs testsuite to fail.  Back it
out until we understand the problem better.

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 src/main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/main.c b/src/main.c
index 65e236be..1614ffbb 100644
--- a/src/main.c
+++ b/src/main.c
@@ -171,7 +171,7 @@ state2:
 state3:
 	state = 4;
 	if (minusc)
-		evalstring(minusc, sflag ? 0 : EV_EXIT);
+		evalstring(minusc, 0);
 
 	if (sflag || minusc == NULL) {
 state4:	/* XXX ??? - why isn't this before the "if" statement */
-- 
1.7.7.rc1

